lib/traject/psulib_config.rb

```diff
@@ -38,7 +38,9 @@
   provide 'reader_class_name', 'Traject::MarcCombiningReader'
   provide 'commit_timeout', '10000'
   provide 'hathi_overlap_path', indexer_settings['hathi_overlap_path'] || '/data/hathitrust_data/'
-  provide 'hathi_overlap_file', indexer_settings['hathi_overlap_file'] || 'final_hathi_overlap.csv'
+  provide 'hathi_mono_overlap_file', indexer_settings['hathi_mono_overlap_file'] || 'final_hathi_mono_overlap.csv'
+  provide 'hathi_multi_overlap_file', indexer_settings['hathi_multi_overlap_file'] || 'final_hathi_multi_overlap.csv'
```

Comment:
```suggestion
```

These two files are produced programmatically by the rake task, so I don't think we need to make them a runtime configuration kinda thing - like the other config things can and do change in different environments and based on what we are trying to do with the traject execution. So, you could just delete them and then in `hash_to_hath` do:

```rb
#custom.rb line 217
     hathi_overlap_csv = "#{settings['hathi_overlap_path']}final_hathi_#{ht_format}_overlap.csv"
```

What do you think?

Reply:
this makes sense, I will try with your suggestion.

Reply:
changes pushed